build.sbt

```diff
@@ -56,7 +56,7 @@ lazy val parent = (project in file("."))
 lazy val `sparkCommons` = (projectMatrix in file("spark-commons"))
   .settings(commonSettings: _*)
   .settings(
-    jacocoReportSettings := commonJacocoReportSettings.withTitle("spark-commons Jacoco Report"),
+    jacocoReportSettings := commonJacocoReportSettings.withTitle(s"spark-commons Jacoco Report - scala: ${scalaVersion.value}"),
```

Comment:
It adds only Scala version info, but Spark versions varies too.

Reply:
Fixed in commit d233375417aef81285e76a666992431dedff0547.